### ISSUE TITLE
Spellcheck offering incorrect corrections

### ISSUE THREAD

**Issue by xNS5** [klinker-apps/messenger-issues#869]
- [X] I have [searched](https://github.com/klinker-apps/messenger-issues/issues), by keyword, for my problem to verify that there aren't any open or closed issues around this behavior.
- [X] I have [searched the help page](https://messenger.klinkerapps.com/help) to verify that it does not have any topics relevant to this issue.
- [X] I have verified that this is something that is reproducible.

**Device with issue:**  Mid 2017 MacBook Pro running Safari v12.1
**Primary device:** Galaxy S8+
**App version:**  Desktop 3.4.3

**Describe the bug:**
Spellcheck finds errors in correctly spelled words, and offers an incorrect word as a replacement.

I've tried quitting the app, and doing a fresh install. Issue still persists. 

[(Pictures)](https://imgur.com/a/hkQgtZt)

**Steps to reproduce:**

1. Type in any kind of contraction (e.g. wasn't, would've weren't)
   Note: It doesn't work for all, but it works on the ones I listed. 

2. See the spellcheck do its thing

3. Left (or right depending on your mouse configuration) click on the underlined word

**Comment by klinker24** [comment]
This is an issue that they have open in electron.

https://github.com/electron/electron/issues/1005

I had used the recommended library, in the past, but moved away from it for various other reasons. 

I have started to build a list of contractions and just include them directly in the app. If you have more than what you listed here, feel free to post them here, or open a PR to add them yourself.

https://github.com/electron/electron/issues/1005

I am going to close this as an upstream issue though. Anyone that comes here can feel free to add any misspellings or issues that they find.
